### PR TITLE
Embed Video Project

### PR DESCRIPTION
**Pull request by Salvador-ON**
First Pull Request

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+</head>
+
+<body>
+    <div class="container">
+        <header class="head">
+            <nav class="menu">
+                <div class="youtubelogo">
+                    <!-- youtube logo -->
+                    <img src="media/ui.png" alt="" id="ui-button">
+                    <img src="media/yt-ico.png" alt="youtube-ico" id="youtube-ico">
```
Comment (pelzolga123) [inline index.html]: Id should be unique, use class instead.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+                    <!-- copy section -->
+
+                    <p id="show-more">Show more</p>
+                </div>
+            </aside>
+        </div>
+
+        <footer>
+            <div class="setting-bar">
+                <img src="media/yt-ico.png" alt="youtube-ico" id="youtube-ico">
```
Comment (pelzolga123) [inline index.html]: Id should be unique, use class instead.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+                <button class="btnf" id="Language-btn"><span id="Language-span">Language: English</span></button>
+                <button class="btnf" id="Location-btn">Location: Netherlands</button>
+                <button class="btnf" id="Restrict-btn">Restricted: Mode Off</button>
+                <button class="btnf" id="History-btn"><span id="History-span">History</span> </button>
+                <button class="btnf" id="Help-btn"><span id="Help-span">Help</span></button>
+            </div>
+
+            <div>
+                <ul class="footer-menu1">
+                    <li id="first-line">About</li>
```
Comment (pelzolga123) [inline index.html]: Id should be unique, use class instead.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+                    <li>Copyright</li>
+                    <li>Creator</li>
+                    <li>Advertise</li>
+                    <li>Developers</li>
+                </ul>
+            </div>
+
+            <div>
+                <ul class="footer-menu2">
+                    <li id="first-line">Terms</li>
```
Comment (pelzolga123) [inline index.html]: Id should be unique, use class instead.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!
Here are the changes, you need to do before this project can be approved:
* [x] Check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows errors in your project.
* [x] Navbar should have a `<button>` tag that represents the search button, right now you have `<input>` tag there ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/search_button_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
* [x] Subscribe should be a button, right now it is just an image ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_subscribe_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

* [x] Pull Request should have a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [x] No ID selectors for multiple elements. Id should always be unique, use class instead.

[OPTIONAL]
* `info-section` will look better if you add a little padding to it.
*  It's a good practice to use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by nriqu322** [comment]
We solve every checkbox indicated above.

-use buttons instead of images.
-correct use of ID and classes
-no errors in validator

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team, You have done a great job with this project. :clap:

**Pull request merged**